lib/components/pop-up/index.tsx

```diff
@@ -25,10 +26,15 @@ const PopUp: React.FC<PopUpProps> = ({
   style,
   manageModal,
   children,
+  size,
 }) => {
   return (
     <>
-      <Modal styleConfig={style} isOpen={manageModal.isOpen} onClose={manageModal.onClose}>
+      <Modal
+        styleConfig={style}
+        isOpen={manageModal.isOpen}
+        onClose={manageModal.onClose}
+        size={size ?? 'lg'}>
```

Comment:
Why did we choose to have our default to be large?

Reply:
I believe it's because the modal on the donation page was size 'lg' and we didn't want to have to change it so we made that the default value. Plus, I'm pretty sure it's chakra ui's default value for modals.